pydantic_settings/main.py

```diff
@@ -26,6 +26,7 @@ class SettingsConfigDict(ConfigDict, total=False):
     env_file_encoding: str | None
     env_ignore_empty: bool
     env_nested_delimiter: str | None
+    env_parse_none: bool
```

Comment:
If desired, we could change this to `env_parse_none_str: str | None` with default as `None`. Users could then define what string value should be parsed (e.g. "null", "void", "None", etc.) into `None`.

Reply:
Yes, let's change it to `env_parse_none_str: str | None`